### ISSUE TITLE
Detailslist: Keyboard users are not able to adjust the width of columns

### ISSUE THREAD

**Issue by Gargi1810** [microsoft/fluentui#21974]
Describe the issue:
Keyboard users are not able to adjust the width of columns of the details list. Keyboard users may find it difficult to adjust the column's width when large information is present in any of the row.

Please provide a reproduction of the issue in a codepen:
you can just try this https://developer.microsoft.com/en-us/fluentui#/controls/web/detailslist/basic

Actual behavior:
Keyboard users are not able to adjust the width of columns of the details list.

Expected behavior:
Keyboard users should be able to adjust the column's width for better visual of row's information in the grid.

I see the bug raised here https://github.com/microsoft/fluentui/issues/17346 but it is not fixed yet

**Comment by Gargi1810** [comment]
Hi team could u plz check this once.

**Comment by micahgodbolt** [comment]
It looks like this feature was added in https://github.com/microsoft/fluentui/pull/20679
